docs/index.md

```diff
@@ -96,6 +153,31 @@ class TestCrud(unittest.TestCase):
 		user = self.your_core_lib.user.create({User.name.key: 'John Dow'})
     self.assertDictEqual(user, self.your_core_lib.user.get(user[User.id.key]))
 ```
+#### Code Explained:
+<b>Writing unit tests for your YourCoreLib class using the unittest framework, alongside Hydra for configuration management.</b>
+- <b>Importing necessary modules:</b>
+
+  - <b>unittest:</b> Standard Python module for writing unit tests.
+  - <b>hydra:</b> Hydra library for managing configurations.
+  - <b>GlobalHydra:</b> A class from Hydra used for global configuration management.
+- <b>Defining a function `get_config()` to use Hydra for loading testing `config.yaml` located under the testing folder.</b>
+  - Clearing any existing Hydra configuration.
+  - Initializing Hydra with a configuration path pointing to `../data/config and using config.yaml`.
+  - Composing configuration, presumably loading it into memory, and returning it.
+  - Creating an instance of YourCoreLib using the configuration obtained from get_config().
+  - Processing outside the test case class, ensuring the configuration is set up before running any tests.
```

Comment:
get_config function is not doing the last two point. remove.

"-Creating an instance of YourCoreLib using the configuration obtained from get_config().
-Processing outside the test case class, ensuring the configuration is set up before running any tests"